과제/week4/src/utils/postUser.ts

```diff
@@ -0,0 +1,29 @@
+import instance from "../lib/axios";
+import { passwordForm } from "../types/dataType";
+
+// 사용자 회원가입
+export async function postUser(formData: passwordForm) {
+  try {
+    const res = await instance.post(`/user`, {
+      username: formData.name,
+      password: formData.password,
+      hobby: formData.hobby,
+    });
```

Comment:
만약 formData의 속성 이름을 API 명세의 필드명과 동일 하게 맞춰놓는다면,

```suggestion
    const res = await instance.post(`/user`, { ...formData });
```

와 같이 해서, 조금더 깔끔하고 간결하게 코드를 작성할 수 있을 것 같습니다!